pkg/tenant/members.go

```diff
@@ -14,8 +14,57 @@ func (s *Server) TenantMemberList(c *gin.Context) {
 	c.JSON(http.StatusNotImplemented, "not implemented yet")
 }
 
+// / TenantMemberCreate adds a new tenant member to the database and returns
+// a 201 StatusCreated response.
+//
+// Route: /tenant/:tenantID/member
 func (s *Server) TenantMemberCreate(c *gin.Context) {
-	c.JSON(http.StatusNotImplemented, "not implemented yet")
+	var (
+		err    error
+		member *api.Member
+	)
+
+	// Bind the user request with JSON and return a 400 response if
+	// biding is not successful.
+	if err = c.BindJSON(&member); err != nil {
+		log.Warn().Err(err).Msg("could not bind tenant member create request")
+		c.JSON(http.StatusBadRequest, api.ErrorResponse("could not bind request"))
+		return
+	}
+
+	// Verify that a member ID does not exist and return a 400 response if
+	// the member id exists.
+	if member.ID != "" {
+		c.JSON(http.StatusBadRequest, api.ErrorResponse("member id cannot be specified on create"))
+		return
+	}
```

Comment:
The member has a `TenantID` field right? For this endpoint we should probably also parse the `:tenantID` from the url and make sure that is set on the `member`. 

```suggestion
	// Verify that a member ID does not exist and return a 400 response if
	// the member id exists.
	if member.ID != "" || member.TenantID != "" {
		c.JSON(http.StatusBadRequest, api.ErrorResponse("member and tenant id cannot be specified on create"))
		return
	}
	
	// TODO: parse :tenantID from URL and add to the member for create.
```

Reply:
The api.Member struct doesn't include TenantID. I can do c.Param("tenantID") to parse the ulid from the URL.

Reply:
Oh, sorry if I'm forgetting, how are we listing members; by OrgID?

Reply:
The api.Member struct currently only has a MemberID.